### PR TITLE
Update report charts

### PR DESCRIPTION
**Pull request by kflemin**
switching reports to Chart js library
Screen Shot 2022-07-13 at 12 29 31 PM

### CODE REVIEW

**Comment by haneslinger** [comment]
@kflemin The charts look good! lmk when you want a more through review.  :)

**Comment by kflemin** [comment]
I think it's ready! @haneslinger

**Comment by RDmitchell** [comment]
It looks great ! Thanks for updating this !

**Review by haneslinger — APPROVED** [review]
looks good to me! thank you. 
Crazy how much of `$scope.yAxisVars` could be reused.
My only comments are about stay print statements and white space changes :P

File: seed/static/seed/js/controllers/inventory_reports_controller.js [hunk 1]
```diff
@@ -140,24 +143,24 @@ angular.module('BE.seed.controller.inventory_reports', [])
           label: $translate.instant('Gross Floor Area'),
           varName: 'gross_floor_area',
           axisLabel: translateAxisLabel('Gross Floor Area', area_units()),
-          axisTickFormat: ',.0f',
-          axisType: 'Measure',
+          // axisTickFormat: ',.0f',
+          // axisType: 'Measure',
```
[the remark this answers is not in the document]
Comment (kflemin) [reply seed/static/seed/js/controllers/inventory_reports_controller.js:147]: these were needed for the old library but unused by chartjs. i'll just delete them

File: seed/static/seed/js/controllers/inventory_plots_controller.js [hunk 2]
```diff
@@ -208,6 +208,8 @@ angular.module('BE.seed.controller.inventory_plots', [])
           chart.data.datasets[0].data = data
           chart.data.datasets[0].labels = labels
           chart.update();
+          console.log("CHART DATA:");
+          console.log(chart.data);
```
[the remark this answers is not in the document]
Comment (kflemin) [reply seed/static/seed/js/controllers/inventory_plots_controller.js]: oops indeed! haha

**Comment by nllong** [comment]
should we merge this down before the big "insights" update?

**Comment by kflemin** [comment]
@nllong this is good to go once the tests finish

**Comment by nllong** [comment]
great, thanks! @kflemin  - can you try to merge this into the latest data aggregation branch? If there are mad conflicts, then please pull in whomever else is needed.

**Pull request merged**